server/models/proxy.py

```diff
@@ -0,0 +1,6 @@
+class ModelProxy(object):
```

Comment:
The return of the `ModelProxy`! 😨 

https://github.com/Cal-CS-61A-Staff/ok/blob/37b27d779129d2e1b63000dbaf4496f2e45fe1d5/server/app/utils.py#L36